error.go

```diff
@@ -27,11 +46,16 @@ func IsInvalidConfig(err error) bool {
 
 var podNotFoundError = microerror.New("pod not found")
 
-// IsNotFound asserts podNotFoundError.
-func IsNotFound(err error) bool {
+// IsPodNotFound asserts podNotFoundError.
+func IsPodNotFound(err error) bool {
 	return microerror.Cause(err) == podNotFoundError
 }
 
+const (
+	releaseNotFoundErrorPrefix = "No such release:"
+	releaseNotFoundErrorSuffix = "not found"
+)
+
```

Comment:
I moved this to have the things related to an error together. I find that easier to grasp what the things are about.